templates/common/infra/bicep/core/ai/cognitiveservices.bicep

```diff
@@ -0,0 +1,38 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
```

Comment:
Do you know what the behavior of this?

Does this:

1. Set `tag` for the resource to empty
2. Or, does not set the `tag` for the resource

Not really concerned, but just good to think about.

Reply:
It means tags is optional.

Reply:
This comment may have been misplaced. What is the behavior when `tag = {}`, and we do: `resource { tag: tag }`?